.changeset/chilly-avocados-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"@snapshot-labs/sx": minor
```

Comment:
```suggestion
"@snapshot-labs/sx": patch
```

changesets bumps `minor` from `0.1.0` to `0.2.0` which is not what we want, we have to force it to `patch` breaking the convention a bit.